examples/pipeline_wavernn/transform.py

```diff
@@ -0,0 +1,59 @@
+import librosa
+import numpy as np
+import torch
+
+
+class Transform():
+    def __init__(self,
+                 sample_rate,
+                 n_fft,
+                 hop_length,
+                 win_length,
+                 num_mels,
+                 fmin,
+                 min_level_db):
+
+        self.sample_rate = sample_rate
+        self.n_fft = n_fft
+        self.hop_length = hop_length
+        self.win_length = win_length
+        self.num_mels = num_mels
+        self.fmin = fmin
+        self.min_level_db = min_level_db
+
+    def load(self, path):
+        waveform = librosa.load(path, sr=self.sample_rate)[0]
+        return torch.from_numpy(waveform)
+
+    def stft(self, y):
+        return librosa.stft(
+            y=y,
+            n_fft=self.n_fft, hop_length=self.hop_length, win_length=self.win_length)
```

Comment:
can you run lint like black on this file to reformat this?

Reply:
Yes, I have reformatted this.